reference/docs-conceptual/developer/help/comment-based-help-keywords.md

```diff
@@ -1,81 +1,148 @@
 ---
-title: "Comment-Based Help Keywords | Microsoft Docs"
-ms.custom: ""
-ms.date: "09/13/2016"
-ms.reviewer: ""
-ms.suite: ""
-ms.tgt_pltfrm: ""
-ms.topic: "article"
-ms.assetid: ab90ec96-77f5-42e3-9c7e-2f4156ec207f
-caps.latest.revision: 6
+title: Comment-Based Help Keywords
+ms.custom:
+ms.date: 06/09/2020
+ms.topic: article
 ---
 # Comment-Based Help Keywords
 
 This topic lists and describes the keywords in comment-based help.
 
 ## Keywords in Comment-Based Help
 
-The following are valid comment-based Help keywords. They are listed in the order in which they typically appear in a Help topic along with their intended use. These keywords can appear in any order in the comment-based Help, and they are not case-sensitive.
+The following are valid comment-based Help keywords. They are listed in the order in which they
+typically appear in a Help topic along with their intended use. These keywords can appear in any
+order in the comment-based Help, and they are not case-sensitive.
 
-Note that the `.ExternalHelp` keyword takes precedence over all other comment-based help keywords. When `.ExternalHelp` is present, the [Microsoft.PowerShell.Commands.GetHelpCommand](/dotnet/api/Microsoft.PowerShell.Commands.gethelpcommand) cmdlet does not display comment-based help, even when it cannot find a help file that matches the value of the keyword.
+Note that the `.ExternalHelp` keyword takes precedence over all other comment-based help keywords.
+When `.ExternalHelp` is present, the
+[Microsoft.PowerShell.Commands.GetHelpCommand](/dotnet/api/Microsoft.PowerShell.Commands.gethelpcommand)
+cmdlet does not display comment-based help, even when it cannot find a help file that matches the
+value of the keyword.
+
+## .Synopsis
 
-`.Synopsis`
 A brief description of the function or script. This keyword can be used only once in each topic.
 
-`.Description`
+## .Description
+
 A detailed description of the function or script. This keyword can be used only once in each topic.
 
-`.Parameter` *\<Parameter-Name>*
-The description of a parameter. You can include a `.Parameter` keyword for each parameter in the function or script.
+## .Parameter \<Parameter-Name>
+
+The description of a parameter. You can include a `.Parameter` keyword for each parameter in the
```

Comment:
Since these key words are typically all capitalized, should the syntax here follow suit? Especially since is is wrapped in inline code? Thesame for later keywords like `.PARAMETER`.

Reply:
I was on the fence about this and you talked me into it. The names are not case-sensitive but the common usage is uppercase. So I changed it.